src/Modules/ModParser.lua

```diff
@@ -3950,6 +3952,9 @@ local specialModList = {
 	},
 	["you count as dual wielding while you are unencumbered"] = { flag("Condition:DualWielding", { type = "Condition", var = "Unencumbered" }) },
 	["dual wielding does not inherently grant chance to block attack damage"] = { flag("Condition:NoInherentBlock") },
+	["inherent attack speed bonus from dual wielding is doubled while wielding two claws"] = {
```

Comment:
This implementation might be incorrect it could be interpreted as double being 2 x 10 = 20 not 1.1 * 1.1 = 1.21 or 21.